Cargo.toml

```diff
@@ -176,13 +177,14 @@ arrow-row = { git = "https://github.com/spiceai/arrow-rs.git", rev = "3e6fcbcb0e
 arrow-select = { git = "https://github.com/spiceai/arrow-rs.git", rev = "3e6fcbcb0e94c411c5e480c77bc830a273f292df" } # spiceai-54.2.1
 arrow-string = { git = "https://github.com/spiceai/arrow-rs.git", rev = "3e6fcbcb0e94c411c5e480c77bc830a273f292df" } # spiceai-54.2.1
 
-datafusion = { git = "https://github.com/spiceai/datafusion.git", rev = "a31aef39aea666f2150c8568f8f5737addc81996" } # spiceai-45
-datafusion-common = { git = "https://github.com/spiceai/datafusion.git", rev = "a31aef39aea666f2150c8568f8f5737addc81996" } # spiceai-45
-datafusion-execution = { git = "https://github.com/spiceai/datafusion.git", rev = "a31aef39aea666f2150c8568f8f5737addc81996" } # spiceai-45
-datafusion-expr = { git = "https://github.com/spiceai/datafusion.git", rev = "a31aef39aea666f2150c8568f8f5737addc81996" } # spiceai-45
+datafusion = { git = "https://github.com/spiceai/datafusion.git", rev = "c118e722ceec90be232d4f2f7683e8b6a81b55b5" } # spiceai-46
+datafusion-common = { git = "https://github.com/spiceai/datafusion.git", rev = "c118e722ceec90be232d4f2f7683e8b6a81b55b5" } # spiceai-46
+datafusion-execution = { git = "https://github.com/spiceai/datafusion.git", rev = "c118e722ceec90be232d4f2f7683e8b6a81b55b5" } # spiceai-46
+datafusion-expr = { git = "https://github.com/spiceai/datafusion.git", rev = "c118e722ceec90be232d4f2f7683e8b6a81b55b5" } # spiceai-46
+datafusion-datasource = { git = "https://github.com/spiceai/datafusion.git", rev = "c118e722ceec90be232d4f2f7683e8b6a81b55b5" } # spiceai-46
 
-datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "1c814422cfbdeb7f7bd2399e3369656710413aa8" } # spiceai-45
-datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "1713e9af2729f686aac46aef63e8c84545083e29" } # spiceai
+datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "a4c752a172904a0c214fc37ab2d62e907eb16c8f" } # spiceai-46
+datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "9c246d91df647597ad076bdb34f8c25eb9754d7c" } # phillip/250425-df-46
```

Comment:
Once this PR is ready to be merged, I'll raise a PR against datafusion-table-providers to merge this and then I'll update the hash here.

Reply:
PR is here: https://github.com/datafusion-contrib/datafusion-table-providers/pull/346

Once I get signoffs, I'll merge the dependent PR and update this one.